### PR TITLE
Merge final release/18.1 into develop

### PR DESCRIPTION
**Pull request by AliSoftware**
- [x] WordPress 18.1 submitted to Google for review
 - [x] Jetpack 18.1 submitted to Google for review

#### Conflict resolution

Got this conflict on `WordPress/build.gradle`, and kept **both** sides.

```
<<<<<<< merge/release-18.1-into-develop
            versionName versionProperties.getProperty("alpha.versionName")
            versionCode versionProperties.getProperty("alpha.versionCode").toInteger()
=======
            buildConfigField "boolean", "RECOMMEND_THE_APP", "true"
>>>>>>> develop
```

### CODE REVIEW

File: WordPress/build.gradle [hunk 1]
```diff
@@ -165,13 +175,19 @@ android {
             applicationIdSuffix ".beta"
             dimension "buildType"
 
+            versionName versionProperties.getProperty("alpha.versionName")
+            versionCode versionProperties.getProperty("alpha.versionCode").toInteger()
+
             buildConfigField "boolean", "RECOMMEND_THE_APP", "true"
```
Comment (AliSoftware) [inline WordPress/build.gradle:181]: (The conflict was there)

File: WordPress/build.gradle [hunk 2]
```diff
@@ -124,6 +124,8 @@ android {
 
     flavorDimensions "app", "buildType"
 
+    def versionProperties = loadPropertiesFromFile(file("${rootDir}/version.properties"))
```
Comment (AliSoftware) [inline WordPress/build.gradle:127]: All those changes in the build.gradle file are from https://github.com/wordpress-mobile/WordPress-Android/issues/15230 (which was merged in the release branch but we didn't merge `release/` to `develop` since, because we didn't do any new beta after that point)

File: Gemfile.lock [hunk 3]
```diff
@@ -1,3 +1,22 @@
+GIT
+  remote: git@github.com:wordpress-mobile/release-toolkit.git
+  revision: 908ff74c45a3ccd022adb028b2b4a2fad296e27a
+  ref: 908ff74c45a3ccd022adb028b2b4a2fad296e27a
```
Comment (AliSoftware) [inline Gemfile.lock:4]: See [comment in the `fastlane/Pluginfile` file](https://github.com/wordpress-mobile/WordPress-Android/pull/15282/files#diff-fc04a28e75ab5e6e1aeb7ac7f0759e9c9d7ef3d650e581c0f355f0e4cfeae9e8R9).

TL;DR: it's expected that we're pointing on a ref and not on a stable version of the toolkit here.

**Review by oguzkocer — APPROVED** [review]
